lib/src/screens/exchange_trade/exchange_confirm_page.dart

```diff
@@ -95,21 +95,39 @@ class ExchangeConfirmPage extends BasePage {
                   ],
                 ),
               ),
-              Flexible(
-                child: Center(
-                  child: Row(
-                    mainAxisAlignment: MainAxisAlignment.center,
-                    children: [
-                      (trade.provider.image?.isNotEmpty ?? false)
-                          ? Image.asset(trade.provider.image, height: 50)
-                          : const SizedBox(),
-                      if (!trade.provider.horizontalLogo)
-                        Padding(
-                          padding: const EdgeInsets.symmetric(horizontal: 8.0),
-                          child: Text(trade.provider.title),
+              Expanded(
+                child: Column(
+                  children: [
+                    if (trade.provider == ExchangeProviderDescription.trocador)
+                    Padding(
+                      padding: const EdgeInsets.only(top: 8.0),
+                      child: Text(
+                       'selected Trocador provider: ${trade.providerName}',
```

Comment:
Missing Localisation 

```shell
dart ./tool/append_translation.dart selected_trocador_provider "selected Trocador provider: {trocador_provider}"
```

Reply:
```suggestion
                       S.of(context).selected_trocador_provider(trade.providerName),
```

After appending the localisation of course :)

Reply:
done